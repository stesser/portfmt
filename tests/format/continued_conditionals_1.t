.if defined(DESTDIR) && !empty(DESTDIR) && !defined(CHROOTED) && \
	!defined(BEFOREPORTMK) && !defined(INOPTIONSMK) && \
	!defined(BEFOREPORTMK) && !defined(INOPTIONSMK)
_FOOBAR=	lasjflas
.endif
<<<<<<<<<
.if defined(DESTDIR) && !empty(DESTDIR) && !defined(CHROOTED) && \
	!defined(BEFOREPORTMK) && !defined(INOPTIONSMK) && \
	!defined(BEFOREPORTMK) && !defined(INOPTIONSMK)
_FOOBAR=	lasjflas
.endif
