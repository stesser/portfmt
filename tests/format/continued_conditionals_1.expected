.if defined(DESTDIR) && !empty(DESTDIR) && !defined(CHROOTED) && \
	!defined(BEFOREPORTMK) && !defined(INOPTIONSMK) && \
	!defined(BEFOREPORTMK) && !defined(INOPTIONSMK)
_FOOBAR=	lasjflas
.endif
